src/Services/MessageService.php

```
@@ -93,9 +94,9 @@ public function paginateMessages(array $query = [], ?int $maxResults = null): Gm
      * @param  array<string, mixed>  $query  Search/filter parameters
      * @param  int  $maxResults  Maximum number of results per page
      * @param  bool  $fullDetails  Whether to fetch full message details
-     * @return \Illuminate\Support\LazyCollection<int, Email> Empty lazy collection (implementation placeholder)
+     * @return LazyCollection<int, Email> Empty lazy collection (implementation placeholder)
      */
-    public function lazyLoadMessages(array $query = [], ?int $maxResults = null, bool $fullDetails = true): \Illuminate\Support\LazyCollection
+    public function lazyLoadMessages(array $query = [], ?int $maxResults = null, bool $fullDetails = true): LazyCollection
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$query'. (undefined)

(UnusedFormalParameter)